packages/teams-js/src/internal/utils.ts

```diff
@@ -338,3 +339,24 @@ export function createTeamsAppLink(params: pages.NavigateToAppParams): string {
   }
   return url.toString();
 }
+
+/**
+ * @hidden
+ * Checks if the Adaptive Card schema version is supported by the host.
+ * @param hostAdaptiveCardSchemaVersion Hosts supported Adaptive Card Version in the runtime.
+ *
+ * @returns true if the Adaptive Card Version is not supported and false if it is supported.
+ */
+export function isHostAdaptiveCardSchemaVersionUnsupported(
```

Comment:
Just curious why we're checking if the version is **Un**supported (rather than supported)? Seems a little counterintuitive to check for a negative condition (and thus return *true*). I just noticed this now (earlier read it as a isSupported check), so wondering if customers might run into the same misunderstanding

Reply:
This check is internal, and customers won't be using this. This will be hidden from the docs